libs/go/aealite/protocols/dialogue.go

```diff
@@ -366,7 +451,15 @@ func (dialogue *Dialogue) isBelongingToADialogue(message AbstractMessage) bool {
 	return result
 }
 
-func InitializeMessage(counterParty Address, selfAddress Address, performative Performative, content []byte, ref [2]string, messageId MessageId, target MessageId) AbstractMessage {
+func InitializeMessage(
+	counterParty Address,
+	selfAddress Address,
+	performative Performative,
+	content []byte,
```

Comment:
You will only partially construct the message, the developer has to then finish populating its field outside the context of this library.